### PR TITLE
Add article to remind current maintained branches and remind infos about PRs, contributions, timelines / Extended support

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add article to remind current maintained branches and remind infos about PRs, contributions, timelines. It's nice to repeat things sometimes to make sure information reaches destination.
| Fixed ticket? |

### CODE REVIEW

File: news/_posts/2023-01-03-178-lts-80.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+icon: icon-warning
+---
+
+Now that PrestaShop 8.0 is out, here is a focus on maintained branches of the PrestaShop project and what to expect.
+
+## Reminder: PrestaShop branching strategy
+
+In [August 2015](https://build.prestashop-project.org/news/2015/introducing-new-branching-model-prestashop/) PrestaShop adopted a new branching model for the project.
+
+The model was refined over the years, for example with [this page on the developer documentation](https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/supported-branches/#bug-fixes-and-patch-branches) about what bugs do qualify for a patch version or the [Backward Compatibility Promise](https://github.com/PrestaShop/ADR/blob/master/0017-backward-compatibility-promise.md) that defines what can changes can be introduced in a version, but we can say the model has lived quite well as the project is still following it in 2023.
```
Comment (0x346e3730) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
The model was refined over the years, for example with [this page on the developer documentation](https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/supported-branches/#bug-fixes-and-patch-branches) about what bugs do qualify for a patch version or the [Backward Compatibility Promise](https://github.com/PrestaShop/ADR/blob/master/0017-backward-compatibility-promise.md) that defines what changes can be introduced in a version, but we can say the model has lived quite well as the project is still following it in 2023.
```

File: news/_posts/2023-01-03-178-lts-80.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Following PrestaShop 8.0.0 release, the `1.7.8.x` branch has entered security-only maintenance phase. As detailed in [the article published in December 2021](https://build.prestashop-project.org/news/2021/prestashop-beyond-1-7/), this means that patch releases of PrestaShop 1.7.8 will only be published if critical bugs are reported or if security fixes are needed.
+
+As a consequence, please do not open a pull request against this branch unless it is a bugfix that solves a critical bug or a security fix. Other Pull Requests will not be accepted.
+
+This maintenance period will end when PrestaShop 9.0.0 is released. When this day comes PrestaShop 1.7 will not be maintained anymore.
+
+### 8.0.x
+
+Branch `8.0.x` is the branch that gave birth to PrestaShop 8.0.0 and the patches that will follow. It was branched out from `develop` branch when [PrestaShop 8.0 freeze](https://build.prestashop-project.org/news/2022/prestashop-feature-freeze-v8/) happened in June 2022.
```
Comment (0x346e3730) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
Branch `8.0.x` is the branch that gave birth to PrestaShop 8.0.0 and the patches that followed. It was branched out from `develop` branch when [PrestaShop 8.0 freeze](https://build.prestashop-project.org/news/2022/prestashop-feature-freeze-v8/) happened in June 2022.
```

**Review by 0x346e3730 — DISMISSED** [review]
Minor modifications

File: news/_posts/2023-01-03-178-lts-80.md (hunk 1 above)
Comment (matks) [reply news/_posts/2023-01-03-178-lts-80.md]: Fixed

File: news/_posts/2023-01-03-178-lts-80.md (hunk 2 above)
Comment (matks) [reply news/_posts/2023-01-03-178-lts-80.md]: No, there is no 8.0 patches today :) . The first patch will be 8.0.1 and it is not released yet. So it is "will follow" ;)

File: news/_posts/2023-01-03-178-lts-80.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+ - 1.7.8.x
+ - 8.0.x
+icon: icon-warning
+---
+
+Now that PrestaShop 8.0 is out, here is a focus on maintained branches of the PrestaShop project and what to expect.
+
+## Reminder: PrestaShop branching strategy
+
+In [August 2015](https://build.prestashop-project.org/news/2015/introducing-new-branching-model-prestashop/) PrestaShop adopted a new branching model for the project.
```
Comment (kpodemski) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
In [August 2015]({{< relref "/news/posts/2015/introducing-new-branching-model-prestashop" >}}) PrestaShop adopted a new branching model for the project.
```

File: news/_posts/2023-01-03-178-lts-80.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+The model was refined over the years, for example with [this page on the developer documentation](https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/supported-branches/#bug-fixes-and-patch-branches) about what bugs do qualify for a patch version or the [Backward Compatibility Promise](https://github.com/PrestaShop/ADR/blob/master/0017-backward-compatibility-promise.md) that defines what changes can be introduced in a version, but we can say the model has lived quite well as the project is still following it in 2023.
+
+## PrestaShop branches in 2023
+
+As of today, January 2023, what are the maintained branches and how do they comply with this model?
+
+On [PrestaShop main repository](https://github.com/prestashop/prestashop) three branches are maintained
```
Comment (kpodemski) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
On [PrestaShop main repository](https://github.com/prestashop/prestashop), three branches are maintained.
```

File: news/_posts/2023-01-03-178-lts-80.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+### 1.7.8.x
+
+Branch `1.7.8.x` is the branch that gave birth to PrestaShop 1.7.8.0 and the patches that followed. As of today last 1.7.8 version is [PrestaShop patch release 1.7.8.8](https://build.prestashop-project.org/news/2022/prestashop-1-7-8-8-maintenance-release/).
```
Comment (kpodemski) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
Branch `1.7.8.x` is the branch that gave birth to PrestaShop 1.7.8.0 and the patches that followed. As of today last 1.7.8 version is [PrestaShop patch release 1.7.8.8]({{< relref "/news/releases/prestashop-1-7-8-8-maintenance-release" >}}).
```

File: news/_posts/2023-01-03-178-lts-80.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+Following PrestaShop 8.0.0 release, the `1.7.8.x` branch has entered security-only maintenance phase. As detailed in [the article published in December 2021](https://build.prestashop-project.org/news/2021/prestashop-beyond-1-7/), this means that patch releases of PrestaShop 1.7.8 will only be published if critical bugs are reported or if security fixes are needed.
```
Comment (kpodemski) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
Following PrestaShop 8.0.0 release, the `1.7.8.x` branch has entered the security-only maintenance phase. As detailed in [the article published in December 2021]({{< relref "/news/posts/2021/prestashop-beyond-1-7" >}}), this means that patch releases of PrestaShop 1.7.8 will only be published if critical bugs are reported or if security fixes are needed.
```

File: news/_posts/2023-01-03-178-lts-80.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+As a consequence, please do not open a pull request against this branch unless it is a bugfix that solves a critical bug or a security fix. Other Pull Requests will not be accepted.
```
Comment (kpodemski) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
Consequently, please do not open a pull request against this branch unless it is a bug fix that solves a critical or security bug. Other pull requests will not be accepted.
```

File: news/_posts/2023-01-03-178-lts-80.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+This maintenance period will end when PrestaShop 9.0.0 is released. When this day comes PrestaShop 1.7 will not be maintained anymore.
```
Comment (kpodemski) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
This maintenance period will end when PrestaShop 9.0.0 is released. When this day comes, PrestaShop 1.7 will not be maintained anymore.
```

File: news/_posts/2023-01-03-178-lts-80.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
+### 8.0.x
+
+Branch `8.0.x` is the branch that gave birth to PrestaShop 8.0.0 and the patches that will follow. It was branched out from `develop` branch when [PrestaShop 8.0 freeze](https://build.prestashop-project.org/news/2022/prestashop-feature-freeze-v8/) happened in June 2022.
```
Comment (kpodemski) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
Branch `8.0.x` is the branch that introduced PrestaShop 8.0.0 and the patches that will follow. It was taken out from the `develop` branch when [PrestaShop 8.0 freeze]({{< relref "content/news/posts/2022/prestashop-feature-freeze-v8" >}}) happened in June 2022.
```

File: news/_posts/2023-01-03-178-lts-80.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+
+Following PrestaShop 8.0.0 release, the `8.0.x` branch is now the supported patch branch. It receives critical bug fixes, security bug fixes, and same-version regression fixes. You can read more about a patch version lifecycle [here](https://build.prestashop-project.org/news/2020/ps17-patch-release-lifecycle/).
```
Comment (kpodemski) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
Following PrestaShop 8.0.0 release, the `8.0.x` branch is now the supported patch branch. It receives critical bug fixes, security bug fixes, and same-version regression fixes. You can read more about a patch version lifecycle [here]({{< relref "/news/posts/2020/ps17-patch-release-lifecycle" >}}).
```

File: news/_posts/2023-01-03-178-lts-80.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+Patches for branch `8.0.x` will be delivered until PrestaShop 8.1.0 is released.
+
+### develop
+
+The `develop` branch is always the next minor or major release. As of today, code changes merged inside this branch will be shipped with next PrestaShop 8 version: PrestaShop 8.1.0 . When PrestaShop 8.1.0 feature freeze happens, a `8.1.x` branch will be created from `develop` and `develop` will become the recipient for changes for the version after PrestaShop 8.1.0 .
```
Comment (kpodemski) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
The `develop` branch is always the next minor or major release. As of today, code changes merged inside this branch will be shipped with the next PrestaShop 8 version: PrestaShop 8.1.0. When PrestaShop 8.1.0 feature freeze happens, an `8.1.x` branch will be created from `develop`, and `develop` will become the recipient for changes for the version after PrestaShop 8.1.0, version 9.0.0.
```

File: news/_posts/2023-01-03-178-lts-80.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+
+You can read more about a minor version lifecycle [here](https://build.prestashop-project.org/news/2020/ps17-minor-release-lifecycle/).
```
Comment (kpodemski) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
You can read more about a minor version lifecycle [here]({{< relref "/news/posts/2020/ps17-minor-release-lifecycle" >}}).
```

File: news/_posts/2023-01-03-178-lts-80.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+
+Most changes to the PrestaShop project should be submitted against this branch.
+
+## Next versions to be released
+
+As explained above, the two next versions of PrestaShop that can be expected are:
+- [Patch version 8.0.1](https://github.com/PrestaShop/PrestaShop/issues/30128), release of the `8.0.x` branch
+- [Minor version 8.1.0](https://github.com/PrestaShop/PrestaShop/issues/30719), release of the `8.1.x` branch not created yet
+
+If a critical bug or a security issue are reported on PrestaShop 1.7.8, a patch version 1.7.8.9 will be released.
```
Comment (kpodemski) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
If a critical bug or a security issue is reported on PrestaShop 1.7.8, a patch version 1.7.8.9 will be released.
```

**Comment by matks** [comment]
@kpodemski All suggestions adresses, can we publish :) ?

File: news/_posts/2023-01-03-178-lts-80.md [hunk 14]
```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title:  "PrestaShop 1.7.8 is now a LTS"
+subtitle: "What to expect for PrestaShop 1.7 and 8"
+date:   2023-01-03
```
Comment (kpodemski) [inline news/_posts/2023-01-03-178-lts-80.md]:
```suggestion
date:   2023-01-05
```

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
@matks after changing the date 👍🏻😀

**Comment by matks** [comment]
@kpodemski I'm not 100% satisfied with my title and subtitle. Do you have suggestions?

**Comment by kpodemski** [comment]
Thanks @matks!

**Pull request merged**